src/components/login/Login.tsx

```diff
@@ -145,81 +151,131 @@ export const Login: React.FunctionComponent = () => {
                   <CircularProgress color="primary" />
                 </div>
               )}
-
               <div className="btm-50 text-center">
-                <Typography variant="h4">{t('common.signIn')}</Typography>
+                {!didSignUp && (
+                  <>
+                    <div className=" btm-20">
```

Comment:
add className="btm-50 for more space
![image](https://user-images.githubusercontent.com/11825561/120865952-de6b8080-c543-11eb-82ad-ba32d085590a.png)